cognite/client/data_classes/contextualization.py

```diff
@@ -844,3 +844,16 @@ def save_predictions(
         raise CogniteException(
             "Extract job is not completed. If the job is queued or running, wait for completion and try again"
         )
+
+
+class ResourceReference(CogniteResource):
```

Comment:
Why does this need to be a CogniteResource?

Reply:
The generic _list returns a CogniteResourceList, might get around that maybe. The smoothest experience would probably be to return the actual resources, and use files/3D apis to retrieve them.

Reply:
Ah now I see.